odoo/addons/base/tests/test_display_name.py

```diff
@@ -12,6 +12,12 @@
     'account_followup.manual_reminder',
 }
 
+IGNORE_COMPUTED_FIELDS = {
```

Comment:
A list of fields to ignore from other modules is a bad practice. The test should work for all of them.

Reply:
I see that the compute method of that field is decorated with onchange, maybe adding a depends could solve the issue.